components/LeagueEntries/LeagueEntries.test.tsx

```diff
@@ -32,11 +43,13 @@ describe('LeagueEntries', () => {
     const leagueEntryPickButton = screen.getByTestId(
       'league-entry-pick-button',
     );
+    const userHistoryPicks = screen.queryByTestId('user-pick-history');
 
     expect(entryStatus).toHaveTextContent('alive');
     expect(leagueEntryContainerCard).toBeInTheDocument();
     expect(leagueEntryNumber).toHaveTextContent('Entry 2');
     expect(leagueEntryPickButton).toHaveTextContent('Change Pick');
+    expect(userHistoryPicks).toBeInTheDocument();
```

Comment:
We could combine line 46 and this line to one line for cleaner code

Reply:
✅ done.